src/modules/features/types.ts

```diff
@@ -38,5 +39,7 @@ export type FeatureSagasConfig = {
 }
 
 export type StateWithFeatures = {
-  features: FeaturesState
+  // Possibly undefined because clients might not have implemented te features module into their dapps.
```

Comment:
typo error.
`// Possibly undefined because clients might not have implemented the features module into their dapps.`

Reply:
Thanks 😄